ui/src/app/pages/workspace/workspace-edit.spec.tsx

```diff
@@ -262,6 +274,83 @@ describe('WorkspaceEdit', () => {
     expect(wrapper.find('[data-test-id="old-cdr-version-warning"]').exists()).toBeFalsy();
   });
 
+  it('does not display the access tier dropdown when multiple tiers are not available', async() => {
+    cdrVersionStore.set( {tiers: [cdrVersionTiersResponse.tiers[0]]});
+
+    const wrapper = component();
+    await waitOneTickAndUpdate(wrapper);
+
+    expect(wrapper.find('[data-test-id="select-access-tier"]').exists()).toBeFalsy();
+  });
+
+  it('enables access tier selection on creation when multiple tiers are available', async() => {
+    workspaceEditMode = WorkspaceEditMode.Create;
+
+    const wrapper = component();
+    await waitOneTickAndUpdate(wrapper);
+
+    const accessTierSelection = wrapper.find('[data-test-id="select-access-tier"]');
+    expect(accessTierSelection.exists()).toBeTruthy();
+
+    // defaults to registered
+    const selectionProps = accessTierSelection.find('select').props();
+    expect(selectionProps.disabled).toBeFalsy();
+    expect(selectionProps.value).toBe(AccessTierShortNames.Registered);
+
+    // when Registered is selected, the CDR Version dropdown lists the registered tier CDR Versions
+    // defaultCdrVersion and altCdrVersion, with defaultCdrVersion selected
+
+    const cdrVersionsSelect = wrapper.find('[data-test-id="select-cdr-version"]').find('select');
+    expect(cdrVersionsSelect.props().value).toBe(defaultCdrVersion.cdrVersionId);
+
+    const cdrVersionSelectOptions: Array<string> = cdrVersionsSelect.children().map(o => o.props().value);
+    expect(cdrVersionSelectOptions).toEqual([defaultCdrVersion.cdrVersionId, altCdrVersion.cdrVersionId]);
+
+    // when Controlled is selected, the CDR Version dropdown lists the (one) controlled tier CDR Version
+
+    // TODO make this test work somehow - I have manually tested the functionality here
```

Comment:
I am not confident I will figure this out in the time that our Controlled Tier schedule will allow.

Reply:
I can take a quick look this afternoon - what is the issue you're currently hitting in the commented out code?

Reply:
Thanks!  The `expect` fails because it sees the default RT CDR Version instead of CT - so my guess is that the change is not registering.